sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/package-info.java

```diff
@@ -1,14 +1,458 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * Package containing classes for creating clients {@link
- * com.azure.security.keyvault.administration.KeyVaultAccessControlAsyncClient} and {@link
- * com.azure.security.keyvault.administration.KeyVaultAccessControlClient} that perform access control operations on
- * Azure Key Vault resources, as well as clients {@link
- * com.azure.security.keyvault.administration.KeyVaultBackupAsyncClient} and {@link
- * com.azure.security.keyvault.administration.KeyVaultBackupClient} that perform backup and restore operations on Azure
- * Key Vault keys.
+ * <a href="https://learn.microsoft.com/azure/key-vault/managed-hsm/">Azure Key Vault Managed HSM</a> is a
+ * fully-managed, highly-available, single-tenant, standards-compliant cloud service that enables you to safeguard
+ * cryptographic keys for your cloud applications using FIPS 140-2 Level 3 validated HSMs.
+ *
+ * <p>The Azure Key Vault Administration client library allows developers to interact with the Azure Key Vault Managed
+ * HSM service from their applications. The library provides a set of APIs that enable developers to perform
+ * administrative tasks such as full backup/restore, key-level role-based access control (RBAC), and account settings
+ * management.
+ *
+ * <p><strong>Key Concepts:</strong>
+ *
+ * <p>What is a Key Vault Access Control Client?
```

Comment:
Wrap the questions in `<strong>` tag in all places
<p><strong>What is a Key Vault Access Control Client?</strong></p>